src/TodoList.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const todoList = [
+    {id:1, title: "Complete lesson one assignment"},
+    {id:2, title: "Practice Leetcode"},
+    {id:3, title: "Do Laundry"},
+    ];
```

Comment:
If you are using VS Code, consider using a code formatter like Prettier to handle indentation for you! It's fine for now while our code is small, but in future weeks indentation may save your life. Make sure to check "format on save" in your VS Code settings once you install something!